src/DynamoCoreWpf/Views/PackageManager/TermsOfUseView.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using Dynamo.ViewModels;
+
+
+namespace Dynamo.PackageManager.UI
+{
+    /// <summary>
+    /// Interaction logic for TermsOfUseView.xaml
+    /// </summary>
+    public partial class TermsOfUseView : Window
+    {
+        public bool AcceptedTermsOfUse { get; set; }
+
+        public TermsOfUseView(TermsOfUseViewModel viewModel)
```

Comment:
Update this constructor to take in the path `string touFilePath`, then give `views:RichTextFile` in the XAML a name, say, `TermsOfUseContent`. Then here you can do: `TermsOfUseContent.File = touFilePath`.